draft-ietf-rtcweb-jsep.xml

```diff
@@ -5909,6 +5920,13 @@ a=rtcp-fb:100 nack pli
           <xref target="RFC4585" />, but not previously handled, and the media
           source SSRC for the packet is found in the outgoing SSRC table,
           deliver the packet to the "m=" line associated with that SSRC.</t>
+		
+          <t>If the packet is of type XR, as defined in <xref target="RFC3611" />,
```

Comment:
Given the bundle does not make XR MTI, I think that this should be prefixed with "If the device supports XR, and ...." ANd make the 3611 ref informative not normative.